content/blog/tina-v-0.69.0.md

```diff
@@ -0,0 +1,164 @@
+2.  Data layer is enabled by default
+3.  Better support for read only tokens
+4.  Cleared and more concise way of working with TinaCMS in CI
+
+To update to the latest version please run
+
+```bash
+yarn add tinacms@latest
+yarn add --dev @tinacms/cli@latest
+```
+
+## New Unified client
+
+TinaCMS has always been missing a "unified way" to query and fetch content. Previously, there was \`StaticRequest\`, "Read only token client", "Experimental Generate client" and "just using fetch". Now all these are amalgamated into one client that can be used on both the frontend and backend.
+
+There are a couple of small changes to be made in order to update.
+
+### Updates in `.tina/schema.{ts,js}`
+
+Instead of passing an `apiURL`, now the clientID, branch and read only token (NEW) will all be configured in the schema. The local url will be used if the `--local` flag is passed.
+
+This will require a change to the schema and the scripts.
+
+```diff
+// .tina/schema.ts
++ import { client } from "./__generated__/client";
+// ...
+const schema = defineSchema({
++    config: {
++        branch: "main",
++        clientID: "***",
++        token: "***",
+    },
+    collections: [
+        // ...
+    ]
+})
+// ...
+- const branch = process.env.NEXT_PUBLIC_VERCEL_GIT_COMMIT_REF
+- const clientId = 'YOUR-CLIENT-ID-HERE'
+- const apiURL =
+-   process.env.NODE_ENV == 'development'
+-     ? 'http://localhost:4001/graphql'
+-    : `https://content.tinajs.io/content/${clientId}/github/${branch}`
+export const tinaConfig = defineConfig({
++  client,
+-  apiURl,
+  schema,
+  // ...
+})
+export default schema
+```
+
+The token must be a wildcard token (`*`) and can be generated from the tina dashboard. [Read more hear](https://tina.io/docs/graphql/read-only-tokens/)
+
+For more information of where to get these values [please read out "going to production docs"](/docs/tina-cloud/connecting-site/#enabling-tina-cloud-in-tinacms)
+
+### Updates to scripts in package.json
+
+We now recommend separating the graphQL server into two separate processes (two separate terminals in development). The scripts should look like this:
+
+```json
+{
+  "scripts": {
+    "dev": "tinacms build --local && next dev",
+    "dev-server": "tinacms server:start",
+    "build": "tinacms build && next build"
+    // ... Other Scripts
+  }
+}
+```
+
+When developing, in the first terminal run `yarn dev-server` and then `yarn dev` in the second.
+
+The old `-c` subcommand can still be used. This will start the dev server and next dev process in the same terminal.
+
+```json
+{
+  "scripts": {
+    "dev": "tinacms server:start \"tinacms build --local && next dev\"",
+    "dev-server": "tinacms server:start",
+    "build": "tinacms build && next build"
+    // ... Other Scripts
+  }
+}
+```
+
+### Updates to generated files (optional)
+
+We now recommend ignoring most of the generated files. This is because `client.ts` and `types.ts` will be generated in CI with `tinacms build`
```

Comment:
```suggestion
We now recommend ignoring most of the generated files. This is because `client.ts` and `types.ts` will be generated in CI when `tinacms build` runs.
```